### DISCUSSION TITLE
How I've upgraded ignite-cli from 7.15.0 to 8.0.0 on Linux

### DISCUSSION THREAD

**Discussion by kabeza** [infinitered/ignite#2231] [General]
I saw the recently upgraded ignite-cli and wanted to update mine on my Linux environment

I executed `sudo npm uninstall -g ignite-cli`
And then `npx ignite-cli new igniteDemo`

But kept getting the 7.15.0 version
![imagen_0879](https://user-images.githubusercontent.com/274856/192793763-4fd056cb-5d32-4e64-8742-22dc17e1491e.jpg)

So I executed again `sudo npm uninstall -g ignite-cli` and then `sudo npm i ignite-cli`
Doing this way worked fine
![imagen_0878](https://user-images.githubusercontent.com/274856/192794069-dc593294-1e4f-42ee-980d-4b0c14646fed.jpg)

Node v16.17.0
Linux ElementaryOS (Ubuntu 20.04 based)

**Comment by frankcalise** [comment]
Thanks for the write up!

Yes, if you had it globally installed, you'll need to upgrade it. 

Alternatively, you may use `npx ignite-cli new PizzaApp` which should just use the latest version at the time of executing the command. This way you won't have to manage and keep globally installed packages up to date.

**Reply by kabeza** [reply]
The thing is that I don't remember installing it globally, but well...
